pkg/systemd/host.css

```diff
@@ -212,6 +212,11 @@ a.disabled {
     word-wrap: break-word;
 }
 
+#system_information_machine_id_row .tooltip-inner {
+    font-family: monospace;
+    max-width: 35ex;
```

Comment:
What is the intention of this?  Can't we just let the tooltip be as large as the browser/bootstrap makes it?  Also, `ex` is for height, and `em` is for width, no?

Reply:
The intention was to make this wide enough to fit the 33 characters of a machine id. Without this, tooltips have a too narrow max width, so that the machine ID doesn't even fit on default zoom.

Indeed I should  use `em` instead of `ex`, this fixes the truncation. Thanks for spotting!

Reply:
> Without this, tooltips have a too narrow max width

Ah, so this overrides an existing style, makes sense.